dubbo-common/src/main/java/org/apache/dubbo/config/nested/TripleConfig.java

```diff
@@ -0,0 +1,254 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.config.nested;
+
+import java.io.Serializable;
+
+/**
+ * Configuration for triple protocol.
+ */
+public class TripleConfig implements Serializable {
+
+    private static final long serialVersionUID = -3682252713701362155L;
+
+    /**
+     * Maximum allowed size for HTTP1 request bodies.
+     * Limits the size of request to prevent excessively large request.
+     * <p>The default value is 8MiB.
+     */
+    private Integer maxBodySize;
+
+    /**
+     * Maximum allowed size for HTTP1 response bodies.
+     * Limits the size of responses to prevent excessively large response.
+     * <p>The default value is 8MiB.
+     */
+    private Integer maxResponseBodySize;
+
+    /**
+     * Set the maximum chunk size.
+     * HTTP requests and responses can be quite large, in which case it's better to process the data as a stream of
+     * chunks.
+     * This sets the limit, in bytes, at which Netty will send a chunk down the pipeline.
+     * <p>The default value is 8MiB.
+     */
+    private Integer maxChunkSize;
+
+    /**
+     * Set the maximum line length of header lines.
+     * This limits how much memory Netty will use when parsing HTTP header key-value pairs.
+     * You would typically set this to the same value as {@link #setMaxInitialLineLength(Integer)}.
+     * <p>The default value is 8KiB.
+     */
+    private Integer maxHeaderSize;
+
+    /**
+     * Set the maximum length of the first line of the HTTP header.
+     * This limits how much memory Netty will use when parsed the initial HTTP header line.
+     * You would typically set this to the same value as {@link #setMaxHeaderSize(Integer)}.
+     * <p>The default value is 4096.
+     */
+    private Integer maxInitialLineLength;
+
+    /**
+     * Set the initial size of the temporary buffer used when parsing the lines of the HTTP headers.
+     * <p>The default value is 16384 octets.
+     */
+    private Integer initialBufferSize;
+
+    /**
+     * The header table size.
+     */
+    private Integer headerTableSize;
+
+    /**
+     * Whether to enable push, default is false.
+     */
+    private Boolean enablePush;
+
+    /**
+     * Maximum concurrent streams.
+     */
+    private Integer maxConcurrentStreams;
+
+    /**
+     * Initial window size.
+     */
+    private Integer initialWindowSize;
+
+    /**
+     * Maximum frame size.
+     */
+    private Integer maxFrameSize;
+
+    /**
+     * Maximum header list size.
+     */
+    private Integer maxHeaderListSize;
+
+    /**
+     * Whether to pass through standard HTTP headers, default is false.
+     */
+    private Boolean passThroughStandardHttpHeaders;
+
+    public TripleConfig() {
+        init();
```

Comment:
The if statement will not be effective. Can we define a default value constant and initialize it directly in the field?

Reply:
Okay, I'll fix it.